spec/lib/diaspora/markdownify_email_spec.rb

```diff
@@ -8,12 +8,12 @@
 
     it 'should autolink a hashtag' do
       markdownified = @html.preprocess("#tag")
-      expect(markdownified).to eq("[#tag](http://localhost:9887/tags/tag)")
+      expect(markdownified).to eq("[#tag](#{AppConfig.url_to('tags/tag')})")
```

Comment:
Doing `include Rails.application.routes.url_helpers` in the toplevel `describe` block should make it available.